Resources/Locale/en-US/reagents/meta/botany.ftl

```diff
@@ -2,22 +2,22 @@ reagent-name-e-z-nutrient = EZ nutrient
 reagent-desc-e-z-nutrient = Give your plants some of those EZ nutrients! Dionas find this delicious.
 
 reagent-name-left4-zed = left-4-zed
-reagent-desc-left4-zed = A cocktail of mutagenic compounds, which cause plant life to become highly unstable.
+reagent-desc-left4-zed = A cocktail of mutagenic compounds that gives nutrients, harms and affects plant life's genome.
 
 reagent-name-pest-killer = pest killer
-reagent-desc-pest-killer = A mixture that targets pests.
+reagent-desc-pest-killer = A mixture that targets pests. While useful it slowly poisons plants with toxins, be careful when using it.
 
 reagent-name-plant-b-gone = plant-B-gone
 reagent-desc-plant-b-gone = A harmful toxic mixture to kill plantlife. Very effective against kudzu.
 
 reagent-name-robust-harvest = robust harvest
-reagent-desc-robust-harvest = A highly effective fertilizer. A botanist's go-to and a diona's best friend.
+reagent-desc-robust-harvest = A highly effective fertilizer, with a limited potency-boosting effect on plants. Be careful with it's usage since using too much has a chance to reduce the plant yield. It has a positive effect on dionas. ```
```

Comment:
```suggestion
reagent-desc-robust-harvest = A highly effective fertilizer, with a limited potency-boosting effect on plants. Be careful with it's usage since using too much has a chance to reduce the plant yield. It has a positive effect on dionas.
```